--- plplot-5.8.0/lib/nn/Makefile.am.orig	2009-12-14 13:28:47.600031053 +0100
+++ plplot-5.8.0/lib/nn/Makefile.am	2009-12-14 13:29:04.234087633 +0100
@@ -24,7 +24,7 @@
 # If using the gcc compiler, this library needs to be built with 
 # the -fno-force-mem -ffloat-store for most reliable performance.
 if cc_is_gcc
-  AM_CFLAGS = -fno-force-mem -ffloat-store
+  AM_CFLAGS = -ffloat-store
 endif
 		
 lib_LTLIBRARIES = libcsironn.la
