--- plplot-5.10.0/cmake/modules/language_support/cmake/CMakeTestDCompiler.cmake.orig	2014-02-12 23:19:33.000000000 +0100
+++ plplot-5.10.0/cmake/modules/language_support/cmake/CMakeTestDCompiler.cmake	2014-03-22 15:08:43.608106139 +0100
@@ -97,7 +97,7 @@
 	OUTPUT_VARIABLE OUTPUT) 
     ELSE(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
       TRY_COMPILE(CMAKE_D_PHOBOS_WORKS ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/testDCompiler.d
-	CMAKE_FLAGS "-DLINK_LIBRARIES=${D_PATH}/lib/libphobos.a"
+	CMAKE_FLAGS "-DLINK_LIBRARIES=${D_PATH}/lib${LIB_SUFFIX}/libphobos2.a"
 	COMPILE_DEFINITIONS "-I${D_PATH}/include -I${D_PATH}/import"
 	OUTPUT_VARIABLE OUTPUT) 
     ENDIF(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
@@ -137,7 +136,7 @@
 	OUTPUT_VARIABLE OUTPUT) 
     ELSE(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
       TRY_COMPILE(CMAKE_D_TANGO_WORKS ${CMAKE_BINARY_DIR} ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/testDCompiler.d
-	CMAKE_FLAGS "-DLINK_LIBRARIES=${D_PATH}/lib/libtango.a;${D_PATH}/lib/libphobos.a"
+	CMAKE_FLAGS "-DLINK_LIBRARIES=${D_PATH}/lib${LIB_SUFFIX}/libtango.a;${D_PATH}/lib${LIB_SUFFIX}/libphobos2.a"
 	COMPILE_DEFINITIONS "-I${D_PATH}/include -I${D_PATH}/import"
 	OUTPUT_VARIABLE OUTPUT) 
     ENDIF(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
--- plplot-5.10.0/cmake/modules/language_support/cmake/Platform/Linux-dmd.cmake.orig	2014-03-22 15:16:28.518086629 +0100
+++ plplot-5.10.0/cmake/modules/language_support/cmake/Platform/Linux-dmd.cmake	2014-03-22 15:10:21.461435366 +0100
@@ -25,7 +25,7 @@
 	#ENDFOREACH(item)
 ENDIF(CMAKE_D_BUILD_DOCS)
 
-SET (CMAKE_D_FLAGS_INIT "-version=Posix ${DSTDLIB_FLAGS} -I$ENV{D_PATH}/include -I$ENV{D_PATH}/import")
+SET (CMAKE_D_FLAGS_INIT "${DSTDLIB_FLAGS} -I$ENV{D_PATH}/include -I$ENV{D_PATH}/import")
 SET (CMAKE_D_FLAGS_DEBUG_INIT "-g ${DDOC_FLAGS}")
 #  SET (CMAKE_D_FLAGS_MINSIZEREL_INIT "-Os ${DDOC_FLAGS}")
 SET (CMAKE_D_FLAGS_RELEASE_INIT "-O -release -inline ${DDOC_FLAGS}")
