--- plplot-5.9.9/cmake/modules/freetype.cmake.orig	2011-10-13 02:43:01.000000000 +0200
+++ plplot-5.9.9/cmake/modules/freetype.cmake	2011-11-19 16:31:29.385484168 +0100
@@ -102,22 +102,6 @@
       )
   endforeach(FONT_ENTRY PL_FREETYPE_FONT_LIST) 
 
-  # Check a couple of fonts actually exists
-  if (EXISTS ${PL_FREETYPE_FONT_DIR}${PL_FREETYPE_SANS})
-    if (EXISTS ${PL_FREETYPE_FONT_DIR}${PL_FREETYPE_SYMBOL})  
-    else (EXISTS ${PL_FREETYPE_FONT_DIR}${PL_FREETYPE_SYMBOL})
-      message("Fonts not found - disabling freetype")
-      set(WITH_FREETYPE OFF CACHE BOOL 
-	"Enable driver options for using freetype library for fonts" FORCE
-	)  
-    endif (EXISTS ${PL_FREETYPE_FONT_DIR}${PL_FREETYPE_SYMBOL})  
-  else (EXISTS ${PL_FREETYPE_FONT_DIR}${PL_FREETYPE_SANS})
-    message("Fonts not found - disabling freetype")
-    set(WITH_FREETYPE OFF CACHE BOOL 
-      "Enable driver options for using freetype library for fonts" FORCE
-      )
-  endif (EXISTS ${PL_FREETYPE_FONT_DIR}${PL_FREETYPE_SANS})
-
 endif (WITH_FREETYPE)
 
 if (WITH_FREETYPE)
