--- plplot-5.8.0/bindings/tk/tkMain.c~	2007-11-18 21:04:01.000000000 +0100
+++ plplot-5.8.0/bindings/tk/tkMain.c	2008-04-02 18:22:34.000000000 +0200
@@ -86,7 +86,7 @@
 /* From tclIntDecls.h */
 
 EXTERN int		Tcl_Import _ANSI_ARGS_((Tcl_Interp * interp,
-				Tcl_Namespace * nsPtr, char * pattern,
+				Tcl_Namespace * nsPtr, CONST char * pattern,
 				int allowOverwrite));
 
 EXTERN Tcl_Namespace *	Tcl_GetGlobalNamespace _ANSI_ARGS_((
