--- plplot-5.3.0/drivers/linuxvga.c.orig	2004-01-19 20:10:19.000000000 +0100
+++ plplot-5.3.0/drivers/linuxvga.c	2004-03-07 02:20:02.895178672 +0100
@@ -35,7 +35,7 @@
 
 static void lxvga_text	(PLStream *pls);
 static void lxvga_graph	(PLStream *pls);
-static void pause	(PLStream *pls);
+static void lxvga_pause	(PLStream *pls);
 
 /* INDENT ON */
 
@@ -158,7 +158,7 @@
 plD_eop_vga(PLStream *pls)
 {
     if (page_state == DIRTY)
-	pause(pls);
+	lxvga_pause(pls);
 
     /* vga_setmode(mode); */
     vga_clear();		/* just clean it */
@@ -255,7 +255,7 @@
 {
     if (pls->graphx == GRAPHICS_MODE) {
 	if (page_state == DIRTY)
-	    pause(pls);
+	    lxvga_pause(pls);
 	vga_setmode(TEXT);
 	pls->graphx = TEXT_MODE;
     }
@@ -278,13 +278,13 @@
 }
 
 /*--------------------------------------------------------------------------*\
- * pause()
+ * lxvga_pause()
  *
  * Wait for a keystroke.
 \*--------------------------------------------------------------------------*/
 
 static void
-pause(PLStream *pls)
+lxvga_pause(PLStream *pls)
 {
     if (pls->nopause) 
 	return;
