--- plplot-5.2.1/lib/csa/csa.c.orig	2003-03-28 16:31:34.000000000 +0100
+++ plplot-5.2.1/lib/csa/csa.c	2003-12-31 17:23:24.671620952 +0100
@@ -32,6 +32,7 @@
 #include <math.h>
 #include <assert.h>
 #include <string.h>
+#include <errno.h>
 #include "version.h"
 #include "csa_internal.h"
 #include "nan.h"
@@ -43,8 +44,6 @@
 #define NMAX_DEF 40
 #define K_DEF 140
 
-extern int errno;
-
 int csa_verbose = 0;
 
 static void csa_quit(char* format, ...)
