diff -ur plplot-5.8.0/cf/octave.ac plplot-5.8.0-oct3/cf/octave.ac
--- plplot-5.8.0/cf/octave.ac	2007-11-18 21:04:07.000000000 +0100
+++ plplot-5.8.0-oct3/cf/octave.ac	2008-04-03 19:45:08.000000000 +0200
@@ -84,10 +84,15 @@
 	# Get Configuration default values for Octave
         AC_MSG_CHECKING([Octave prefix])
         PL_OCTAVE_CONFIG_INFO(OCTAVE_PREFIX, prefix)
+        [OCTAVE_PREFIX=`echo $OCTAVE_PREFIX | sed 's|^[^/]\+/|/|'`]
         AC_MSG_RESULT($OCTAVE_PREFIX)
 
         AC_MSG_CHECKING([Octave m dir])
-	if test -n "`echo $OCTAVE_VERSION | sed 's/^2.1//p;d;q'`" ; then
+	if test -n "`echo $OCTAVE_VERSION | sed 's/^3.0//p;d;q'`" ; then
+            # Octave version 3.0
+            PL_OCTAVE_CONFIG_INFO(OCTAVE_M_DIR, localfcnfiledir)
+            [OCTAVE_M_DIR=`echo $OCTAVE_M_DIR | sed 's|^[^/]\+/|/|'`]
+        elif test -n "`echo $OCTAVE_VERSION | sed 's/^2.1//p;d;q'`" ; then
             # Octave version 2.1
             PL_OCTAVE_CONFIG_INFO(OCTAVE_M_DIR, localfcnfiledir)
         else
@@ -106,7 +111,11 @@
         AC_MSG_RESULT($OCTAVE_M_DIR)
 
         AC_MSG_CHECKING([Octave oct dir])
-	if test -n "`echo $OCTAVE_VERSION | sed 's/^2.1//p;d;q'`" ; then
+	if test -n "`echo $OCTAVE_VERSION | sed 's/^3.0//p;d;q'`" ; then
+            # Octave version 3.0
+            PL_OCTAVE_CONFIG_INFO(OCTAVE_OCT_DIR, localoctfiledir)
+            [OCTAVE_OCT_DIR=`echo $OCTAVE_OCT_DIR | sed 's|^[^/]\+/|/|'`]
+        elif test -n "`echo $OCTAVE_VERSION | sed 's/^2.1//p;d;q'`" ; then
             # Octave version 2.1
             PL_OCTAVE_CONFIG_INFO(OCTAVE_OCT_DIR, localoctfiledir)
         else
