--- plplot-5.9.9/bindings/ocaml/CMakeLists.txt.orig	2012-07-19 13:21:54.994034873 +0200
+++ plplot-5.9.9/bindings/ocaml/CMakeLists.txt	2012-07-19 13:23:28.724031567 +0200
@@ -40,7 +40,7 @@
         elseif(DEP STREQUAL "qsastime")
           set(internal_LIB_DIR "qsastime")
         endif()
-        set(ocaml_STATIC_FLAGS ${ocaml_STATIC_FLAGS} -cclib ${CMAKE_BINARY_DIR}/lib/${internal_LIB_DIR}/lib${DEP}.a)
+        set(ocaml_STATIC_FLAGS ${ocaml_STATIC_FLAGS} -cclib ${CMAKE_BINARY_DIR}/lib${LIB_SUFFIX}/${internal_LIB_DIR}/lib${DEP}.a)
       elseif(DEP STREQUAL "general")
         set(ocaml_STATIC_FLAGS ${ocaml_STATIC_FLAGS})
       else()
--- plplot-5.9.9/cmake/modules/ocaml.cmake~	2011-10-13 02:43:01.000000000 +0200
+++ plplot-5.9.9/cmake/modules/ocaml.cmake	2012-07-19 13:29:30.017352156 +0200
@@ -130,7 +130,7 @@
 
   # Installation follows the Debian ocaml policy for want of a better
   # standard.
-  set(OCAML_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib/ocaml
+  set(OCAML_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/ocaml
     CACHE PATH "install location for ocaml files"
     )
 
