--- plplot-5.9.9/scripts/CMakeLists.txt.orig	2011-10-13 02:43:01.000000000 +0200
+++ plplot-5.9.9/scripts/CMakeLists.txt	2011-11-19 07:31:37.394399567 +0100
@@ -30,8 +30,8 @@
   if(PLD_pstex)
     set(scripts_SCRIPTS ${scripts_SCRIPTS} pstex2eps)
   endif(PLD_pstex)
-  if (NOT ${scripts_SCRIPTS} STREQUAL "")
+  if (NOT "${scripts_SCRIPTS}" STREQUAL "")
     install(PROGRAMS ${scripts_SCRIPTS} DESTINATION ${BIN_DIR})
-  endif (NOT ${scripts_SCRIPTS} STREQUAL "")
+  endif (NOT "${scripts_SCRIPTS}" STREQUAL "")
 endif(HAVE_BASH)
 
