--- plplot-5.3.1/bindings/python/Makefile.am.orig	2005-01-09 22:27:24.763930256 +0100
+++ plplot-5.3.1/bindings/python/Makefile.am	2005-01-09 22:27:59.969578184 +0100
@@ -68,7 +68,7 @@
 	$(PYTHON) makedocstrings.py plplotcmodule_p_single.c plplotcmodule_single.c
 
 # Must have "exec" in the name.
-ourexecpythondir = $(pythondir)
+ourexecpythondir = $(pyexecdir)
 
 libplplot = $(top_builddir)/src/libplplot$(LIB_TAG).la
 
