--- plplot-5.2.1/utils/Makefile.am.orig	2003-03-06 13:40:36.000000000 +0100
+++ plplot-5.2.1/utils/Makefile.am	2004-02-01 01:32:54.920491019 +0100
@@ -25,10 +25,11 @@
 AM_CPPFLAGS = $(INCLTDL) -I../bindings/tcl $(TCLINCCMD) $(ITCLINCCMD)
 
 if enable_tcl
-bin_PROGRAMS = plrender pltek pltcl
+pltclprog = pltcl
 else
-bin_PROGRAMS = plrender pltek
+pltclprog =
 endif
+bin_PROGRAMS = plrender pltek $(pltclprog)
 
 plrender_SOURCES = plrender.c
 plrender_LDFLAGS = $(RPATH) -no-undefined
